sdks/python/apache_beam/examples/inference/sklearn_mnist_classification.py

```diff
@@ -60,7 +62,7 @@ def parse_known_args(argv):
   """Parses args for the workflow."""
   parser = argparse.ArgumentParser()
   parser.add_argument(
-      '--input_file',
+      '--input',
```

Comment:
let's update the markdown at: https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/inference

to reflect this change. can be a separate pr. 
@rszper , @rezarokni are there any documentations/notebooks where we provide complete command line to run examples and might need to fix this?

Thanks.

Reply:
Not in the documentation that I created. @AnandInguva has been working on notebooks, but I'm not sure if they would be related to this change.

Reply:
thanks!

Reply:
+1 with @rszper. The markdown seems fine with the current implementation as well.